### PR TITLE
validate coco

### PR DESCRIPTION
**Pull request by santicevic**
# Validate [COCO](https://solscan.io/token/74DSHnK1qqr4z1pXjLjPAVi8XFngZ635jEVpdkJtnizQ)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/studwud/status/1691816644387959243
- Coingecko/ CMC URL (If available): N/A

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -760,3 +760,4 @@ NINJA TURTLES,NINJA,DFrJxDoLMYt6bNYeNe8Wrjzj2UPUSLZLEMMYBLuTKcTk,8,https://arwea
 Sdrive Coin,SCOIN,5qKDWkBejLtRh1UGFV7e58QEkdn2fRyH5ehVXqUYujNW,9,https://arweave.net/O5kTUpi6mNl-FWY2zq01MqK0qkwRKy3O5cyaoosyJwM,true
 sunSBR,sunSBR,sunSBR8hRLZy13HCHdrT3hjJZVRpkVfXF3QaaJYjxZ2,6,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/sunSBR8hRLZy13HCHdrT3hjJZVRpkVfXF3QaaJYjxZ2/logo.svg,true
 Threshold Bitcoin,tBTC,6DNSN2BJsaPFdFFc1zP37kkeNe4Usc1Sqkzr9C9vPWcU,8,https://bafybeidlxvypapcw2gbs7f2mazbub75kiphz4fgrmljgdjoiswmx32zkri.ipfs.dweb.link/,true
+COCO Token,COCO,74DSHnK1qqr4z1pXjLjPAVi8XFngZ635jEVpdkJtnizQ,5,https://d1emnl92bfxoe0.cloudfront.net/location-loot/icon/coco.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:763]:
```suggestion
COCO Token,COCO,74DSHnK1qqr4z1pXjLjPAVi8XFngZ635jEVpdkJtnizQ,5,https://shdw-drive.genesysgo.net/EV1ARo89dwRzR1kv7JMr7V97qrcXjffkcwEuNHMJfJmz/COCO_icon.png,true
```

**Review by Timoon21 — COMMENTED** [review]
Change to matches on-chain metadata

File: validated-tokens.csv [hunk 2]
```diff
@@ -760,3 +760,4 @@ NINJA TURTLES,NINJA,DFrJxDoLMYt6bNYeNe8Wrjzj2UPUSLZLEMMYBLuTKcTk,8,https://arwea
 Sdrive Coin,SCOIN,5qKDWkBejLtRh1UGFV7e58QEkdn2fRyH5ehVXqUYujNW,9,https://arweave.net/O5kTUpi6mNl-FWY2zq01MqK0qkwRKy3O5cyaoosyJwM,true
 sunSBR,sunSBR,sunSBR8hRLZy13HCHdrT3hjJZVRpkVfXF3QaaJYjxZ2,6,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/sunSBR8hRLZy13HCHdrT3hjJZVRpkVfXF3QaaJYjxZ2/logo.svg,true
 Threshold Bitcoin,tBTC,6DNSN2BJsaPFdFFc1zP37kkeNe4Usc1Sqkzr9C9vPWcU,8,https://bafybeidlxvypapcw2gbs7f2mazbub75kiphz4fgrmljgdjoiswmx32zkri.ipfs.dweb.link/,true
+COCO Token,COCO,74DSHnK1qqr4z1pXjLjPAVi8XFngZ635jEVpdkJtnizQ,5,https://shdw-drive.genesysgo.net/EV1ARo89dwRzR1kv7JMr7V97qrcXjffkcwEuNHMJfJmz/COCO_icon.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:763]:
```suggestion
COCO Token,COCO,74DSHnK1qqr4z1pXjLjPAVi8XFngZ635jEVpdkJtnizQ,5,https://shdw-drive.genesysgo.net/EV1ARo89dwRzR1kv7JMr7V97qrcXjffkcwEuNHMJfJmz/COCO_icon.png,true
```

**Review by Timoon21 — COMMENTED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation
- [ ] Token is already live and trading on Jupiter

I will approve as soon as token will be tradable on jupiter

**Comment by santicevic** [comment]
@Timoon21 It is tradable on jupiter now

**Pull request merged**